docker-compose.yml

```diff
@@ -31,6 +31,7 @@ services:
     image: postgres:11
     environment:
     - POSTGRES_DB=kvstore
+    - POSTGRES_HOST_AUTH_METHOD=trust
```

Comment:
This eventually will have security ramifications if left in.  I feel like we need more documentation or perhaps a different workflow around this...

Reply:
Ok I will write more about the postgres in the readme. For now when we show this repo to other people helping on the project, I think as long as this work for local testing we should be fine..? Maybe when we decide we can successfully deploy this software we can update the workflow.